index.html

```diff
@@ -85,13 +83,15 @@ <h1>I'm Adeyemi Glad to see you!</h1>
               </div>
             </div>
             <div class="social-media-icon">
-              <li>
-                <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a>
-                <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin_icon.png" alt="Linkedin"></a>
-                <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
-                <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
-                <a class="handwe" href="#angelist.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
-              </li>
+              <ul>
+                <li>
+                  <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a>
+                  <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin_icon.png" 
+                alt="Linkedin"></a>
+                  <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
+                  <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
+                  <a class="handwe" href="#angelist.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
+              </ul>            
```

Comment:
I salute your tenacity, you have done great implementing the changes requested by the previous reviewers, especially, using `li` tags here. However,the layout could be better. For example, the `li` opening tag has no closing tag.

Here's what that section might look like:

```
<ul>
     <li>
          <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a >
     </li>
     <li>
          <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin_icon.png" 
                alt="Linkedin"></a>
     </li>
     <li>
          <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
     </li>
     <li>
          <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
     </li>
     <li>
          <a class="handwe" href="#angelist.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
     </li>
</ul>    
```

Kindly implement this and your project will be good to go.